include/ccf/historical_queries_interface.h

```diff
@@ -79,6 +79,14 @@ namespace ccf::historical
     virtual void set_default_expiry_duration(
       ExpiryDuration seconds_until_expiry) = 0;
 
+    /** EXPERIMENTAL: Set the tracking of deletes on missing keys for historical
+     * queries.
+     *
+     * This is experimental but setting this to true ensures that the `tx_diff`
+     * available in index handlers can observe deleted values.
+     */
```

Comment:
Is it safe to call after indexing has begun? If yes, does it do what a user might expect (start tracking deletes from there)? A mention of this in the docstring would be nice.